pkg/controller/persistentvolume/types.go

```diff
@@ -25,6 +25,13 @@ import (
 	"k8s.io/kubernetes/pkg/client/cache"
 )
 
+const (
+	// A PV created specifically for one claim must contain this annotation in order to bind to the claim.
+	// The value must be the name of the claim being bound to.
+	// This is an experimental feature and likely to change in the future.
+	createdForKey = "persistent-volume-provisioning.experimental.kubernetes.io/created-for"
```

Comment:
This whole PV feature has been wordy ... but I can type "persistent" very
quickly now, so I've got that going for me.

I'll change it to your suggestion :)

On Wed, Sep 23, 2015 at 5:46 PM, Tim Hockin <notifications@github.com>
wrote:

> In pkg/controller/persistentvolume/types.go
> <https://github.com/kubernetes/kubernetes/pull/14249#discussion_r40261952>
> :
>
> > @@ -25,6 +25,13 @@ import (
> >  	"k8s.io/kubernetes/pkg/client/cache"
> >  )
> >
> > +const (
> > +	// A PV created specifically for one claim must contain this annotation in order to bind to the claim.
> > +	// The value must be the name of the claim being bound to.
> > +	// This is an experimental feature and likely to change in the future.
> > +	createdForKey = "persistent-volume-provisioning.experimental.kubernetes.io/created-for"
>
> that's pretty wordy. how about
> volume.experimental.kubernetes.io/provisioned-for
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14249/files#r40261952>.
>